apicast/src/oauth/keycloak.lua

```diff
@@ -237,9 +237,7 @@ function _M:authorize(service, client)
   end
 
   local url = resty_url.join(self.config.authorize_url, ngx.var.is_args, ngx.var.args)
-  local res = http_client.get(url)
-
-  _M.respond_and_exit(res.status, res.body, res.headers)
+  ngx.redirect(url)
```

Comment:
The relevant section of the RFC is https://tools.ietf.org/html/rfc6749#section-3.1